### DISCUSSION TITLE
Can Linux reduce startup time of evcxr camparing to Windows?

### DISCUSSION THREAD

**Discussion by baiguoname** [evcxr/evcxr#313] [Q&A] [answered]
The startup time is mainly caused by compiling, [there is a article](https://medium.com/aedm/windows-wsl-and-linux-a-build-time-benchmark-for-java-rust-go-and-node-43188a9c77d6) saying that Linux compile time is much less than Windows. So the startup time in Linux could be much less than that in Windows, am I right?

**Comment by davidlattimore** [comment, marked as answer]
Wow, that's surprising how much slower Windows is at compiling Rust code. I wonder what's causing that. To answer your question, yes, that would almost certainly make evcxr slower, both to start and to execute each bit of code. Although I think startup time is to some extent dominated by rust-analyzer warming up, which may have different performance characteristics than compiling Rust code.
